src/core/src/bootstrap/Constants.py

```diff
@@ -310,9 +311,10 @@ class EnvLayer(EnumBackport):
     # This ordering ensures that the most important information is preserved in the case of patch object truncation
     PackageClassificationOrderInStatusReporting = {
         PackageClassification.SECURITY: 1,
-        PackageClassification.CRITICAL: 2,
-        PackageClassification.OTHER: 3,
-        PackageClassification.UNCLASSIFIED: 4
+        PackageClassification.SECURITY_ESM: 2,
+        PackageClassification.CRITICAL: 3,
```

Comment:
Please reorder to Critical, Security, Security_ESM. This order went in when I was away on parental and should not have been this way.

Reply:
Done